sdk/src/main/scala/com/horizen/SidechainState.scala

```diff
@@ -37,6 +38,8 @@ class SidechainState private[horizen] (stateStorage: SidechainStateStorage, over
   },
   s"Specified version is invalid. ${stateStorage.lastVersionId.map(w => bytesToVersion(w.data)).getOrElse(version)} != ${version}")
 
+  private var withdrawalEpochInfo : WithdrawalEpochInfo = WithdrawalEpochInfo(0, 0)
```

Comment:
Each time we start the Node it will lead to setting `withdrawalEpochInfo` to initial value.
State should not keep `withdrawalEpochInfo` itself. Previous value should be retrieved from historyStorage.